scripts/longtermeffects.lua

```diff
@@ -36,94 +31,81 @@ local function onEffectAddStart_new(rEffect)
 	elseif rEffect.sUnits == "day" then
 		rEffect.nDuration = rEffect.nDuration * 14400;
 	end
+
 	rEffect.sUnits = "";
 end
 
-local function resetInit_new()
-	-- De-activate all entries
-	for _,v in pairs(CombatManager.getCombatantNodes()) do
-		DB.setValue(v, "active", "number", 0);
-	end
-
-	-- Clear GM identity additions (based on option)
-	CombatManager.clearGMIdentity();
-
+function onCustomCombatReset()
 	-- Reset the round counter (bmos changed this to 0 instead of 1)
-	DB.setValue("combattracker.round", "number", 0);
-
-	CombatManager.onCombatResetEvent();
+	DB.setValue(CombatManager.CT_ROUND, "number", 0);
 end
```

Comment:
Handle reset as a registered action instead of a copy/paste/modified override, safer.